apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/PDFService.kt

```diff
@@ -83,6 +86,26 @@ class PDFService(
 
         return brev
     }
+
+    companion object {
+        // Kombinerer en liste med PDF-er til én enkelt PDF, og bevarer rekkefølgen fra listen.
+        fun kombinerPdfListeTilEnPdf(pdfListe: List<Pdf>): Pdf {
+            val pdfMerger = PDFMergerUtility()
+            val finalPdf = Loader.loadPDF(pdfListe.first().bytes)
+
+            pdfListe.drop(1).forEach { pdf ->
+                val sourcePdf = Loader.loadPDF(pdf.bytes)
+                pdfMerger.appendDocument(finalPdf, sourcePdf)
+                sourcePdf.close()
+            }
+
+            val out = ByteArrayOutputStream()
+            finalPdf.save(out)
+            finalPdf.close()
+
+            return Pdf(out.toByteArray())
+        }
+    }
```

Comment:
Ville kanskje ikke hatt dette i servicen. For meg høres dette mer ut som en `PdfHelper` eller `PdfUtil`.